src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/tests/latest/test_iot_digitaltwin_commands.py

```diff
@@ -0,0 +1,104 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=too-many-statements
+
+from azure.cli.testsdk import ScenarioTest
+
+
+class IoTDigitalTwinTest(ScenarioTest):
+
+    def test_iot_digitialtwin(self):
+        self.kwargs.update({
+            'endpoint': 'https://provider.azureiotrepository-test.com',
+            'name': 'clitestrepo',
+            'name2': 'clitestrepo-updated',
+            'user_role': 'Reader',
+            'user_role2': 'Admin'
+        })
+
+        # Test 'az iot digitaltwin repository create'
+        repo = self.cmd('iot digitaltwin repository create -e "{endpoint}" -n {name}',
+                        checks=[self.exists('id'),
+                                self.exists('provisioningState'),
+                                self.check('status', 'Provisioned')]).get_output_in_json()
+
+        # Test 'az iot digitaltwin repository get-provision-status'
+        self.cmd('iot digitaltwin repository get-provision-status -e "{endpoint}" -r {} -s {}'.format(repo['id'], repo['provisioningState'], **self.kwargs),
```

Comment:
Once you have updated `self.kwargs` you do not need to use it in `.format()`

`self.cmd()` internally calls `.format()` with the `kwargs`

Please see the test documentation i sent earlier

Reply:
Updated